cri/criservice.go

```diff
@@ -0,0 +1,104 @@
+package cri
+
+import (
+	criv1alpha1 "github.com/alibaba/pouch/cri/v1alpha1"
+	servicev1alpha1 "github.com/alibaba/pouch/cri/v1alpha1/service"
+	criv1alpha2 "github.com/alibaba/pouch/cri/v1alpha2"
+	servicev1alpha2 "github.com/alibaba/pouch/cri/v1alpha2/service"
+	"github.com/alibaba/pouch/daemon/config"
+	"github.com/alibaba/pouch/daemon/mgr"
+
+	"github.com/sirupsen/logrus"
+)
+
+// RunCriService start cri service if pouchd is specified with --enable-cri.
+func RunCriService(daemonconfig *config.Config, containerMgr mgr.ContainerMgr, imageMgr mgr.ImageMgr, stopCh chan error) {
+	var err error
+
+	defer func() {
+		stopCh <- err
+		close(stopCh)
+	}()
+	if !daemonconfig.IsCriEnabled {
+		return
+	}
+	switch daemonconfig.CriConfig.CriVersion {
+	case "v1alpha1":
+		runv1alpha1(daemonconfig, containerMgr, imageMgr)
+	case "v1alpha2":
+		runv1alpha2(daemonconfig, containerMgr, imageMgr)
```

Comment:
If user doesn't specify the CRI version, use "v1alpha2" by default.

Reply:
OK, I'll change it.

Reply:
> If user doesn't specify the CRI version, use "v1alpha2" by default.

Yes, I think this is reasonable to set a default one. And throw out a line of related log.